helm/temporal-worker-controller/templates/rbac.yaml

```
@@ -0,0 +1,278 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .Release.Name }}-manager-role
+  namespace: {{ .Values.namespace }}
+  labels:
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/created-by: temporal-worker-controller
+    app.kubernetes.io/part-of: temporal-worker-controller
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  - events
+  - serviceaccounts
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Release.Name }}-temporal-role
```

Cluster roles are global/not scoped to the helm release's k8s namespace, so we should include the release namespace in the name in order to prevent conflicts if there are multiple installations of the controller in the same k8s cluster.

```suggestion
  name: {{ .Release.Name }}-{{ .Release.Namespace }}-manager-role
```